src/module/keyboard-layouts/sv-latin.js

```diff
@@ -0,0 +1,20 @@
+//SPDX-FileCopyrightText: 2021 Martin (fohswe#8355), GH: plutoneld
+//
+// SPDX-License-Identifier: MIT
+
+export const svLatin = {
```

Comment:
should probably be `svLatin1` instead, unless I am missing something? Also the file should be called accordingly.